Websites/planet.webkit.org/config.ini

```diff
@@ -9,17 +9,17 @@
 # owner_name: Your name
 # owner_email: Your e-mail address
 name = Planet WebKit
-link = http://planet.webkit.org/
-owner_name = Brent Fulgham
-owner_email = bfulgham@apple.com
+link = https://planet.webkit.org/
+owner_name = WebKit Admin
+owner_email = webkit-infrastructure-admins@group.apple.com
 
 # cache_directory: Where cached feeds are stored
 # new_feed_items: Number of items to take from new feeds
 # log_level: One of DEBUG, INFO, WARNING, ERROR or CRITICAL
 # feed_timeout: number of seconds to wait for any given feed
 cache_directory = cache
-new_feed_items = 6
-log_level = DEBUG
+new_feed_items = 500
```

Comment:
Please add a comment explaining how this number was chosen.

Reply:
This number has been set on the server since it was brought up in 2017. I suspected it was copied as is from the older server. The new_feed_items setting limits the number of feed that shows up from an site. My best guess is we wanted to display as many feed as possible from any of the subscribed sites, and thus a large number was chosen.